meteor/sofie-server/data-access/repositories/interfaces/part-repository.ts

```diff
@@ -2,5 +2,6 @@ import { Part } from '../../../model/entities/part'
 
 export interface PartRepository {
 	getParts(segmentId: string): Promise<Part[]>
+	save(part: Part): Promise<void>
```

Comment:
On the `RundownRepository`, the matching method is named `saveRundown`. A similar naming scheme should be used in both places. I'd suggest renaming this to `savePart`. Same applies to `SegmentRepository`.